### PR TITLE
Setup xo and conform to the coding style

### PR DESCRIPTION
**Pull request by uiureo**
I'm sorry #3 doesn't conform to your coding style. Added xo to automate it.

### CODE REVIEW

File: test.js [hunk 1]
```diff
@@ -7,20 +8,25 @@ it('should return the name of named functions', function () {
 });
 
 it('anonymous', function () {
+	/* eslint-disable */
```
Comment (sindresorhus) [inline test.js]: Why?

Comment (uiureo) [inline test.js]: To test a function without space before parenthesis.

```js
assert.equal(fnName(function() {}), null);
```

It produces an error.
```js
  11:30  error  Missing space before function parentheses  space-before-function-paren
```

Do you have a better idea?

Comment (shinnn) [inline test.js]: Like https://github.com/kevva/download/blob/ab37da7408655bbdf87c2c4e186738c92a8707d7/test/test.js#L17, `eslint-disable-line` would make it simpler.

```js
assert.equal(fnName(function() {}), null); // eslint-disable-line
```

Comment (uiureo) [inline test.js]: Looks good!

**Comment by sindresorhus** [comment]
Thanks :)

**Comment by uiureo** [comment]
:raised_hands:

**Pull request closed without merging**